src/wintoastlib.cpp

```diff
@@ -32,24 +32,26 @@
 #pragma comment(lib, "shlwapi")
 #pragma comment(lib, "user32")
 
-#ifdef NDEBUG
-#    define DEBUG_MSG(str)                                                                                                                 \
-        do {                                                                                                                               \
-        } while (false)
-#else
-#    define DEBUG_MSG(str)                                                                                                                 \
-        do {                                                                                                                               \
-            std::wcout << str << std::endl;                                                                                                \
-        } while (false)
-#endif
-
 #define DEFAULT_SHELL_LINKS_PATH L"\\Microsoft\\Windows\\Start Menu\\Programs\\"
 #define DEFAULT_LINK_FORMAT      L".lnk"
 #define STATUS_SUCCESS           (0x00000000)
 
+bool gDebugOutputEnabled = false;
```

Comment:
```suggestion
#ifdef NDEBUG
static bool gDebugOutputEnabled = false;
#else
static bool gDebugOutputEnabled = true;
#endif
```
- Keep the current `NDEBUG` behavior.
- I don't know about the prefix. Looking at other static variables, they use PascalCase without a prefix.